src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --second-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.logo_link {
+  display: inline-block;
+}
+
+.logo_moyo {
+  height: 100%;
+  width: 100%;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+  margin-right: 50px;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: #000;
+  display: inline-block;
+  height: 60px;
```

Comment:
Both `height` and `line-height` are not needed here. With `line-height` you set the height of text to 60px, so if the parent item contains single line of text and has no vertical margins/paddings/borders, it will already be 60px high. So `height` is redundant here.

Reply:
removed